.github/workflows/post-apiview.yml

```
@@ -3,6 +3,10 @@ name: After APIView
 on:
   check_suite:
     types: [completed]
+    
+permissions:
+  pull-requests: write
```

```suggestion
  issues: write
```

For adding a comment, I think you want `issues:write` instead.

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token